contrib/benchmarking_nd4j/.mvn/wrapper/maven-wrapper.properties

```diff
@@ -0,0 +1 @@
+distributionUrl=https://repo1.maven.org/maven2/org/apache/maven/apache-maven/3.5.2/apache-maven-3.5.2-bin.zip
```

Comment:
We should probably update this to match the suggested maven version for the rest of DL4J - Or remove the maven wrapper entirely.

Reply:
Removed.